File: data_structure/array/reverse_a_string/reverse_string.js
function reverse_string (string) {
    let reverse = "";
    for(let i=string.length -1 ; i>=0 ; i--){
        reverse = reverse + string[i];
        // console.log(reverse)
    }
    return reverse;
}

const test1 = "abcde";

console.log(reverse_string(test1));
// reverse_string(test1)
// console.log(test1.length)

File: data_structure/array/reverse_a_string/reverse_string.go
package main

import (
	"fmt"
)

func reverse(str string) string{
	if (len(str) < 2 ) {
		return "Can't reverse string"
	} else {
		revesre_str := "";
		for i := len(str) - 1; i >= 0; i-- {
			revesre_str = revesre_str + string(str[i]);
		}
		return revesre_str;
	} 
}

func main () {
	test1 := "abcde";
	fmt.Println(reverse(test1));
}